build/index.md

```diff
@@ -28,56 +30,74 @@ and much improved functionality that can be powerful tools for improving your
 builds' performance or reusability of your Dockerfiles, and it also introduces
 support for complex scenarios.
 
-## Docker Build features
+Docker Build is way more than the `docker build` command and is not only about
+packaging your code, it's a whole ecosystem of tools and features that support
+you not only with common workflow tasks but also provides you with support for
+more complex and advanced scenarios:
+
+## Building your images
 
-Docker Build is way more than your `docker build` command and is not only about packaging your code, it’s a whole ecosystem of tools and features that support you not only with common workflow tasks but also provides you with support for more complex and advanced scenarios.
-Here’s an overview of all the use cases with which Build can support you:
+### Packaging your software
```

Comment:
I quite liked the old style which did this as bullet points - I think the headers look a bit dense without a lot of content around them. The list style will definitely work a lot better as we rework the explanations and point them towards the new content we're working on.

Reply:
It's just easier to quickly look at headers because you have them available in the sidebar:

![image](https://user-images.githubusercontent.com/1951866/189922285-9ff7616e-ae1f-464f-bcd6-578c5c462062.png)

And also search engines make them referenced.

Reply:
Ah sounds good then :smile: